src/routes/languages/+page.svelte

```diff
@@ -7,26 +7,66 @@
 
 <Stripe>The Languages of Agma Schwa</Stripe>
 <section>
-    <h3 class="text-left">Naturalistic Conlangs</h3>
+    <h3 class="text-left">Conlangs from the world of Djogawa Djororx</h3>
+    
+    <p>These are the languages spoken by the dog people ("pups") in the canon of the <i>In Pursuit of Bez</i> universe, 
+        including the ongoing comic <i><a href="/comic"><strong>Djogawa Djororx</strong></a></i>:
```

Comment:
```suggestion
        including the ongoing comic <a href="/comic"><em>Djogawa Djororx</em></a>:
```
I think just italic is enough.